app/src/main/java/com/example/immadisairaj/codeforces/InfoActivity.java

```diff
@@ -120,33 +120,33 @@ public void showInfo(Info info) {
 
         TextView textView = findViewById(R.id.tv_name_info);
         if (result.getFirstName() != null)
-            textView.setText("Name : " + result.getFirstName() + " " + result.getLastName());
+            textView.setText(getString(R.string.Name) + result.getFirstName() + " " + result.getLastName());
         else
-            textView.setText("Name : Not Provided by user");
+            textView.setText(getString(R.string.Not_Provided));
```

Comment:
can you use it as a two separate resources something like
`Name : ` for one and `Not Provided by user` as another resource

Reply:
Done